app/controllers/tweets_controller.rb

```diff
@@ -11,4 +11,27 @@ def show
     @tweet = Tweet.find(params[:id])
     @user = @tweet.user
   end
+
+  # GET /tweets/new
+  def new
+    @tweet = Tweet.new
+  end
+
+  # POST /tweets
+  def create
+    @tweet = Tweet.new(tweet_params)
+    @tweet.user_id = current_user.id
```

Comment:
Simple

以下のように書けます。

```suggestion
    @tweet = Tweet.new(tweet_params.merge(user: current_user))
```